api/v1beta1/hyperconverged_types.go

```diff
@@ -315,6 +321,15 @@ type LiveMigrationConfigurations struct {
 	AllowPostCopy *bool `json:"allowPostCopy,omitempty"`
 }
 
+// VirtualMachineOptions holds the cluster level information regarding the virtual machine.
+type VirtualMachineOptions struct {
+	// DisableFreePageReporting disable the free page reporting of
+	// memory balloon device https://libvirt.org/formatdomain.html#memory-balloon-device.
+	// This will have effect only if AutoattachMemBalloon is not false and the vmi is not
+	// requesting any high performance feature (dedicatedCPU/realtime/hugePages), in which free page reporting is always disabled.
+	DisableFreePageReporting bool `json:"disableFreePageReporting,omitempty"`
```

Comment:
@fossedihelm - the defaults e2e test is failing. Maybe we need to add also here?

e.g.
```suggestion
	// +kubebuilder:default=true
	// +default=true
	//		DisableFreePageReporting bool `json:"disableFreePageReporting,omitempty"`
```

Reply:
Another option is the `omitempty`. maybe that signals that since missing == `false`, then when we removes this field, it's like setting it to `false`?

Reply:
Similar working example in this file is the `EnableCommonBootImageImport` field.

Reply:
Thanks for the catch, I think you are right!